src/vm/functions/mod.rs

```diff
@@ -267,6 +272,30 @@ pub fn lookup_reserved_functions(name: &str, version: &ClarityVersion) -> Option
             SetVar => SpecialFunction("special_set-var", &database::special_set_variable),
             Map => SpecialFunction("special_map", &sequences::special_map),
             Filter => SpecialFunction("special_filter", &sequences::special_filter),
+            BuffToIntLe => NativeFunction(
+                "native_buff_to_int_le",
+                NativeHandle::SingleArg(&conversions::native_buff_to_int_le),
+                // TODO: Create a dedicated cost function for this case.
```

Comment:
just want to double check if we want to merge these "todos" as well

Reply:
I don't think there's yet a way to represent cost functions for Clarity2.  If not, then @gregorycoppola please create issues to do so, and tag them as `stacks-2.1`.  Thanks!

Reply:
@pavitthrap I had previously merged the todos but @kantai asked to expand them.
@jcnelson I added https://github.com/blockstack/stacks-blockchain/issues/2717